Mathlib/MeasureTheory/Function/LpSeminorm/CompareExp.lean

```diff
@@ -47,7 +47,7 @@ theorem eLpNorm'_le_eLpNormEssSup_mul_rpow_measure_univ {q : ℝ} (hq_pos : 0 <
     eLpNorm' f q μ ≤ eLpNormEssSup f μ * μ Set.univ ^ (1 / q) := by
   have h_le : (∫⁻ a : α, ‖f a‖ₑ ^ q ∂μ) ≤ ∫⁻ _ : α, eLpNormEssSup f μ ^ q ∂μ := by
     refine lintegral_mono_ae ?_
-    have h_nnnorm_le_eLpNorm_ess_sup := coe_nnnorm_ae_le_eLpNormEssSup f μ
+    have h_nnnorm_le_eLpNorm_ess_sup := coe_enorm_ae_le_eLpNormEssSup f μ
```

Comment:
I'm not sure where this lemma appears, but shouldn't it be called just `enorm_ae...` instead of `coe_enorm_ae...`?